test/src/xquery/fn.xql

```diff
@@ -21,4 +41,17 @@ declare
 function fnt:normalize-unicode($normalization-form as xs:string) {
     count(string-to-codepoints(normalize-unicode($fnt:composed, $normalization-form))),
     count(string-to-codepoints(normalize-unicode($fnt:decomposed, $normalization-form)))
-};
\ No newline at end of file
+};
+
+declare
```

Comment:
I don't have a test for that which you know from our above discussion. However this does fix the issue, which you could confirm by testing yourself.